project2/Styles.js

```diff
@@ -0,0 +1,39 @@
+import { StyleSheet } from "react-native";
+import Constants from "expo-constants";
+
+export default StyleSheet.create({
+  appContainer: {
+    flex: 1,
+    backgroundColor: "#AAAAAF",
+    paddingTop: Constants.statusBarHeight,
+  },
+  row: {
+    padding: 20,
+  },
+  listText: {
+    color: "blue",
+  },
+  searchInput: {
+    backgroundColor: "#FFFFFF",
+    borderWidth: 1,
+    borderColor: "black",
+    minWidth: 200,
+    marginTop: 20,
+    marginBottom: 20,
+    marginHorizontal: 20,
+    paddingHorizontal: 10,
+    paddingVertical: 5,
+    borderRadius: 3,
+  },
+  imageWrapper: {
+    alignItems: "center",
+  },
+  image: {
+    width: 200,
+    height: 300,
+  },
+  scroll: {
+    marginTop: 10,
+    marginBottom: 10,
+  },
+});
```

Comment:
Given the size on the app, can just put all styles in one place. Personally, I'd rather the styles were all in one common place so I can easily find properties that are common to multiple components and generalise them but YMMV. Means the file can get quite big, but it's all a tradeoff.